### PR TITLE
Larger comment edits

### PR DESCRIPTION
**Pull request by rudeboybert**
Hey @ismayc any chance you put 👀 on the final revisions for the JOSE article detailed in #121? The big remaining TODO is making the manuscript in `paper.Rmd` shorter by moving content to the package source itself (to the README in our case). I've made the bulk of the transferring of content, but some fresh eyes to make sure that both `paper.Rmd` and the README still respectively flow would be appreciated. Do you have the bandwidth these days?

### CODE REVIEW

**Comment by ismayc** [comment]
I will try to take a look by end of day on Monday

File: vignettes/paper.md [hunk 1]
```diff
@@ -211,508 +200,42 @@ ggplot(evals, aes(x = age, y = score, color = ethnicity)) +
...
-ggplot(score_model_points, aes(x = age, y = residual)) +
-  geom_point() +
-  labs(x = "Age", y = "Residual")
-```
 
-\begin{figure}
+# Repository README
 
-{\centering \includegraphics[width=1\linewidth]{/Users/rudeboybert/Documents/moderndive/moderndive/vignettes/paper_files/figure-latex/residuals-2-1} 
+In the GitHub repository README, we present an in depth discussion of six features of the `moderndive` package:
```
Comment (ismayc) [inline vignettes/paper.md:215]:
```suggestion
In the GitHub repository README, we present an in-depth discussion of six features of the `moderndive` package:
```

File: vignettes/paper.Rmd [hunk 2]
```diff
@@ -193,349 +182,35 @@ ggplot(evals, aes(x = age, y = score, color = ethnicity)) +
...
 
-## 2. Outputs as tibbles
 
-The second common student question:
 
-> "How do we extract the values in the regression table?"
+# Repository README
 
-While one might argue that extracting the intercept and slope coefficients can be "simply" done using `coefficients(score_model)`, what about the standard errors? For example, a Google query of "_how do I extract standard errors from lm in R_" yielded results from [the R mailing list](https://stat.ethz.ch/pipermail/r-help/2008-April/160538.html){target="_blank"} and from [Cross Validated](https://stats.stackexchange.com/questions/27511/extract-standard-errors-of-coefficient-linear-regression-r){target="_blank"} suggesting we run:
+In the GitHub repository README, we present an in depth discussion of six features of the `moderndive` package:
```
Comment (ismayc) [inline vignettes/paper.Rmd:190]:
```suggestion
In the GitHub repository README, we present an in-depth discussion of six features of the `moderndive` package:
```

File: vignettes/paper.Rmd [hunk 3]
```diff
@@ -193,349 +182,35 @@ ggplot(evals, aes(x = age, y = score, color = ethnicity)) +
...
-```
+1. Focus less on p-value stars, more confidence intervals
+2. Outputs as tibbles
+3. Produce residual analysis plots from scratch using `ggplot2`
+4. A quick-and-easy Kaggle predictive modeling competition submission!
+5. Visual model selection: plot parallel slopes & interaction regression models
+6. Produce metrics on the quality of regression model fits
 
-We argue that this task shouldn't be this hard, especially in an introductory statistics setting. To rectify this, the three `get_regression_*` functions  all return data frames in the tidyverse-style tibble (tidy table) format [@R-tibble]. Therefore you can extract columns using the `pull()` function from the `dplyr` package [@R-dplyr]:
+Furthermore, we discuss the inner-workings of the `moderndive` pacakges, including
```
Comment (ismayc) [inline vignettes/paper.Rmd:199]:
```suggestion
Furthermore, we discuss the inner-workings of the `moderndive` package:
```

**Review by ismayc — COMMENTED** [review]
@rudeboybert I made just a few tweaks here to `paper.Rmd` that will require a rebuild for the PDF. We certainly have one of the more impressive README's now 😄

**Comment by rudeboybert** [comment]
@ismayc agreed! but that's how JOSE wants it!

**Pull request merged**